easy_my_coop/models/operation_request.py

```diff
@@ -215,12 +215,14 @@ def get_share_update_mail_template(self):
                             False)
 
     def send_share_trans_mail(self, sub_register_line):
-        cert_email_template = self.get_share_trans_mail_template()
-        cert_email_template.send_mail(rec.partner_id_to.id, False)
+        for rec in self:

```

Comment:
i don't agree with this modification for two reasons :
- you use self instead of rec (it has no implication but anyway it's wrong)
- the fonction is called from another one where a self.ensure_one() is called. So there is no reason to loop in this self.